src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
+@import 'blocks/card';
+@import 'blocks/stars';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: $font-text;
+  font-weight: 400;
```

Comment:
You don't need that, because it's default value in browser

Reply:
removed